### PR TITLE
Update Troubleshooting section in Kuma Docs 

### PR DESCRIPTION
**Pull request by mindy**
Added instructions to fix two issues I ran into while setting up my local environment:
1. KumaScript macros not evaluating
2. Scraping missing documents

### CODE REVIEW

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -76,14 +76,52 @@ them to your user on the host system::
...
+
+    docker-compose logs web kumascript
+    docker-compose logs web
 
 If you have more problems running Kuma, please:
 
 #. Paste errors to `pastebin`_.
 #. Start a thread on `Discourse`_.
-#. After you email dev-mdn, you can also ask in `IRC`_.
+#. After you email dev-mdn, you can also ask in `IRC`_ or the #mdn-dev Slack
```
Comment (peterbe) [inline docs/troubleshooting.rst]: Whilst you're at it, you can remove the mention of IRC entirely.

File: docs/troubleshooting.rst [hunk 2]
```diff
@@ -76,14 +76,52 @@ them to your user on the host system::
...
+
+
+Fixing a 404 error on a specific page
+-------------------------------------
+
+This most likely happens because the document isn't in the database yet, so it 
+needs to be scraped. Run the command 
+``./manage.py scrape_document [url of document]``, e.g.::
+    
+    ./manage.py scrape_document https://developer.mozilla.org/en-US/docs/Glossary/HTML
```
Comment (peterbe) [inline docs/troubleshooting.rst]: I think you should demonstrate the docker-compose version of this instead. 
```suggestion
    docker-compose exec web ./manage.py scrape_document https://developer.mozilla.org/en-US/docs/Glossary/HTML
```

File: docs/data.rst [hunk 3]
```diff
@@ -68,6 +68,7 @@ A local user can be promoted to staff with the command::
 
     ./manage.py ihavepower username --password=password
 
+.. _add-wiki-doc:
```
Comment (peterbe) [inline docs/data.rst]: what is this?

Comment (mindy) [reply docs/data.rst]: This is a reference for the link on line 110 in `docs/troubleshooting.rst`

File: docs/troubleshooting.rst (hunk 1 above)
Comment (mindy) [reply docs/troubleshooting.rst]: 👍 Removed

**Pull request merged**